README.md

```diff
@@ -250,6 +250,20 @@ A curated list of awesome Java frameworks, libraries and software.
 * [documents4j](http://documents4j.com) - API for document format conversion using third-party converters such as MS Word.
 * [jOpenDocument](http://www.jopendocument.org/) - Processes the OpenDocument format.
 
+## Formal Verification
+
+*Formal-methods tools: proof assistants, model checking, symbolic execution etc.*
+
+* [CATG](https://github.com/ksen007/janala2) - A concolic unit testing engine for Java programs. Automatically generates unit tests for Java programs using formal methods.
```

Comment:
1. Please remove the indefinite articles, e.g.

"A concolic unit testing engine"

"Concolic unit testing engine"

2. It is implied that the frameworks, tools, libraries, books and so on are Java-centric. Therefore please remove the explicit phrases when possible, e.g. 

"Concolic unit testing engine for Java programs"

"Concolic unit testing engine"